src/components/PathwayCourse/PathwayCards/index.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { breakpoints } from "../../../theme/constant";
+import Chip from "@mui/material/Chip";
+import {
+  Container,
+  Box,
+  Grid,
+  Card,
+  Button,
+  CardContent,
+  Typography,
+} from "@mui/material";
+import { dateTimeFormat } from "../../../constant";
+
+const PathwayCards = (props) => {
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+
+  const language = {
+    hi: "Hindi",
+    en: "English",
+    mr: "Marathi",
+  };
+  const { userEnrolledClasses } = props;
+
+  return (
+    <>
+      <Container maxWidth="xl">
+        <Typography mb={2} mt={2} variant="h5">
+          Upcoming Classes
+        </Typography>
+
+        <div
+          className="pathway-enrolledClass-cards"
+          style={{
+            display: "flex",
+            maxWidth: "100%",
+            overflowX: "scroll",
+          }}
+        >
+          {userEnrolledClasses?.slice(0, 3).map((item) => {
+            return (
+              <Grid item xs={12} sm={4} md={4}>
+                <Card style={{ minWidth: "300px", margin: "10px" }}>
+                  {item.type == "batch" ? (
+                    <Box sx={{ borderTop: 5, color: "ForestGreen" }} />
+                  ) : (
+                    <Box sx={{ borderTop: 5, color: "darkblue" }} />
+                  )}
```

Comment:
The JavaScript convention is to use `===` when the types on the left and right are expected to be the same (in this case both strings).  Also, as a nitpick, it's good to be consistent with capitalization (e.g., darkblue is lowercase here so forestgreen should be as well.)  Finally, try to use the conditional operator only for what's different:
```suggestion
                        <Box
                          sx={{
                            borderTop: 5,
                            color: item.type === "batch" ? "forestgreen" : "darkblue",
                          }}
                        />
```